data/meetups.yml

```diff
@@ -371,6 +371,17 @@ locations:
     organizers:
     - organizer: Tom Zellman
       profileImage: http://photos2.meetupstatic.com/photos/member/d/a/c/c/member_91316012.jpeg
+  - location: Columbus, Ohio
+    url: http://www.meetup.com/ember-columbus/
+    lat: 39.9833
+    lng: 82.9833
```

Comment:
I think this is supposed to be a negative value for longitude. Otherwise you end up on the map in China :)